### PR TITLE
Made some changes in Intro

### PR DESCRIPTION
**Pull request by SiminaB**
Wanted to have the introduction be a bit clearer about what deep learning is and how it falls into the midst of the "usual methods." Please check for accuracy.

### CODE REVIEW

**Comment by agitter** [comment]
Thanks!  I'm travelling now but will look at this soon if @cgreene doesn't get to it first.

Can you please look into the reported merge conflicts in `02_intro.md` in the meantime?  And remove the `.gitattributes` file?

**Comment by cgreene** [comment]
From a quick glance at the conflicts + network ( https://github.com/greenelab/deep-review/network ) I think this came off an early commit. I can see what I can do with git to rebase onto the current master.

**Comment by SiminaB** [comment]
@agitter Do you want to keep the table I started in https://github.com/SiminaB/deep-review/blob/master/tables.md as a separate file or should I add it to the intro? I wasn't sure if it would be helpful to have all the tables in one file for now due to issues like numbering etc. (Haven't made the pull request for it yet because of that.)

**Comment by agitter** [comment]
@SiminaB I suggest we place the table where it fits best and manually number it for the initial pull request.  A glossary table could fit well in the intro or at the end.

We can make another pull request later to add automatic numbering once that is ready.

**Comment by SiminaB** [comment]
Alright, sounds good! I just made that change.

**Comment by michaelmhoffman** [comment]
Perhaps add more abbreviations from #617? Some of them are not deep learning specific but they might still help, especially for someone who jumps into the middle of the paper.

File: sections/02_intro.md [hunk 1]
```diff
@@ -23,31 +23,47 @@ algorithms have produced comparable or higher accuracy than previous
...
-or more specific tasks. Deep learning does many of the same things as more
-familiar approaches. Like a clustering algorithm, it can build features that
-describe recurrent patterns in data. Like a regression approach, deep learning
-methods can predict some output. However, deep learning methods combine both of
-these steps. When sufficient data are available, these methods construct
+or more specific tasks. 
+
+Deep learning does many of the same things as more
+familiar machine learning approaches. In particular, deep learning approaches can be used both in *supervised*
+applications - where the goal is to accurately predict one or more labels or outcomes associated with each
```
Comment (agapow) [inline sections/02_intro.md]: Is there an emdash vs endash issue, i.e. -- vs -? Raised this earlier and not sure how it resolved.

File: sections/02_intro.md [hunk 2]
```diff
@@ -68,12 +84,44 @@ large. New neural network approaches are also well-suited for addressing
 distinct challenges. For example, neural networks structured as autoencoders or
 as adversarial networks require no labels and are now regularly used for
 unsupervised tasks. In this review, we do not exhaustively discuss the different
-types of deep neural network architectures. A recent book from Goodfellow et al.
+types of deep neural network architectures; an overview of the principal terms used
+herein is given in Table 1. A recent book from Goodfellow et al.
 [@url:http://www.deeplearningbook.org/] covers these in detail. Finally, the
 larger datasets now available are also sufficient for fitting the many
```
Comment (agapow) [inline sections/02_intro.md]: Already mentioned above.

File: sections/02_intro.md [hunk 3]
```diff
@@ -68,12 +84,44 @@ large. New neural network approaches are also well-suited for addressing
...
+| Autoencoder (AE) | NN that sets the outer layer to be similar to the input layer, used for example in dimension reduction |
+| Generative neural network | NN approach that uses models trained to generate data similar to the collected data, leading to smaller number of parameters |
+| Generative adversarial network (GAN) | Generative NN approach that uses two networks, one that generates samples from training data and one that discriminates between generated and training data |
+| Restricted Bolzmann machine (RBM) | Generative NN that forms the building block for many DL approaches, having a single input layer and a single hidden layer, with no connections between the nodes within each layer |
+| Deep belief network (DBN) | Generative NN with several hidden layers, which can be obtained from combining multiple RBMs |
+
+While deep learning shows increased flexibility over other machine learning approaches, as 
+seen in the remainder of this review, it requires large training sets in order to fit
+the hidden layers, as well as accurate labels for the supervised learning applications.
+This is one of the primary drivers for its usefulness in some areas of
```
Comment (agapow) [inline sections/02_intro.md]: This sentence feels a bit wrong: shouldn't be singular, "usefulness" isn't quite the right word. How about: "for these reasons, deep learning has recently become popular within biomedicine while having little adoption in other fields."

Comment (SiminaB) [reply sections/02_intro.md]: I meant to say that even within biomedicine, the "usefulness" ("popularity" is fine too) is not the same across all areas, since the successful use of DL needs large training sets in general and accurate labels for supervised DL applications in particular.

File: sections/02_intro.md [hunk 4]
```diff
@@ -23,31 +23,47 @@ algorithms have produced comparable or higher accuracy than previous
 best-in-class methods that required years of extensive customization, and
 specialized implementations are now being used at industrial scales.
 
+Deep learning approaches grew from research in neural networks.
```
[the remark this answers is not in the document]
Comment (SiminaB) [reply sections/02_intro.md]: That's fine with me.

File: sections/02_intro.md [hunk 5]
```diff
@@ -23,31 +23,47 @@ algorithms have produced comparable or higher accuracy than previous
...
 Neural networks were first proposed in 1943 [@doi:10.1007/BF02478259] as a model
 for how our brains process information. The history of neural networks is
 interesting in its own right [@doi:10.1103/RevModPhys.34.135]. In neural
 networks, inputs are fed into a hidden layer, which feeds into one or more
-hidden layers, which eventually produce an output layer. The neural networks
+hidden layers, which eventually produce an output layer.
```
[the remark this answers is not in the document]
Comment (SiminaB) [reply sections/02_intro.md]: "Link to?" "Connect with?"

File: sections/02_intro.md (hunk 1 above)
Comment (SiminaB) [reply sections/02_intro.md]: Just opened a new issue https://github.com/greenelab/deep-review/issues/695 to ask about that.

Comment (SiminaB) [reply sections/02_intro.md]: Looks like it should the em dash cf https://github.com/greenelab/deep-review/issues/592 https://github.com/greenelab/deep-review/issues/695

File: sections/02_intro.md (hunk 2 above)
Comment (SiminaB) [reply sections/02_intro.md]: @agapow Thanks for your comments! I made changes in response to them, with the exception of this, since I was a bit confused about what you meant.

**Comment by cgreene** [comment]
I have reviewed. Based on the changes in the intro (the rest appear to be due to the move to the new manubot, I think this is ready to go). Note that #632 defined LSTM for the first time in the categorize section. With this new table, we can use the LSTM abbreviation.
